draft-terbu-sd-jwt-vc.md

```diff
@@ -71,88 +139,117 @@ Credentials are cryptographically signed statements about a Subject, typically t
+private claims. Private claims are not relevant for this specification
+due to the openness of the three-party-model. Since SD-JWTs are based on
+JWTs, this specification aims to express the basic Verifiable Credential
+data model purely through JWT Claim Sets, using registered claims while
+allowing Issuers to use additional registered claims, as well as new or
+existing public claims, to make statements about the Subject of the
 Verifiable Credential.
 
-## Requirements Notation and Conventions
+# Native JWT Representation of Verifiable Credentials
 
-The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
-"SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this
-document are to be interpreted as described in RFC 2119 [@!RFC2119].
+## Overview
 
-## Terms and Definitions
+This section provides guidance on how to use JSON [@!RFC8259] claimsets
+with JWT [@!RFC7519] registered claims to construct a JWT that can be
+mapped to a verifiable credential. This section also describes how to
+use content types and token types to distinguish different
+representations of verifiable credentials.
 
-This specification uses the terms "Holder", "Issuer", "Verifier", defined by
-[@!I-D.ietf-oauth-selective-disclosure-jwt].
+This representation relies on claims registered in the [IANA JSON Web
+Token Claims
+Registry](https://www.iana.org/assignments/jwt/jwt.xhtml#claims)
+whenever possible.
 
-Verifiable Credential (VC):
-:  An Issuer-signed assertion with claims about a Subject.
+Implementers using this representation SHOULD NOT use `vc+ld+json` as an
+input.
 
-SD-JWT-based Verifiable Credential (SD-JWT VC):
-: A Verifiable Credential encoded using the Issuance format defined in
-[@!I-D.ietf-oauth-selective-disclosure-jwt].
+### Credential Header
 
-Unsecured payload of an SD-JWT VC:
-: A JSON object containing all selectively disclosable and non-selectively disclosable claims
-of the SD-JWT VC. The unsecured payload acts as the input JSON object to issue
-an SD-JWT VC complying to this specification.
+`typ` MUST use the media type `vc+jwt`.
 
-Status Provider:
-: An entity that provides status information (e.g. revocation) about a Verifiable Credential.
+Example of credential metadata (decoded JWT header):
 
-# Scope
+```json
+{
+  "kid": "https://example.edu/issuers/14#key-0",
+  "alg": "ES256",
+  "typ": "vc+jwt"
+}
+```
 
-* This specification defines
-  - Data model and media types for Verifiable Credentials based on SD-JWTs.
-  - Validation and processing rules for Verifiers and Holders.
+### Credential
+
+Example of a credential (decoded JWT payload):
+
+```json
+{
+  "iss": "https://example.edu/issuers/14",
+  "sub": "1234567890",
+  "name": "John Doe",
+  "iat": 1516239022,
+  "urn:example:claim": true
+}
+```
 
-# Use Cases
+NOTE: The `vc` and `vp` claims MUST NOT be present when the content
+type header parameter is set to `credential-claims-set+json`.
```

Comment:
typ or cty? I don't understand this sentence. Given line 211 states `typ` is `vc+jwt`.